client/src/main/java/com/kritsit/casetracker/client/domain/services/Administrator.java

```diff
@@ -93,4 +93,42 @@ public int deleteUser(Map<String, Object> inputMap){
         return connection.getInspectors();
     }
 
+    public InputToModelParseResult editUser(Map<String, Object> inputMap) {
+        if (inputMap == null || inputMap.isEmpty()) {
+            logger.debug("InputMap empty or null. Aborting");
+            return new InputToModelParseResult(false, "Required information missing");
+        }
+        
+        logger.info("Edit user {}", inputMap.get("username"));
+        
+        InputToModelParseResult result = new InputToModelParseResult(true);
+        
+        for(Map.Entry<String, Object> entry : inputMap.entrySet()){
```

Comment:
Can this not be refactored into its own method, as it is the same as the addUser section?